tests/utils.go

```diff
@@ -548,6 +611,170 @@ func CreateHostPathPvWithSize(osName string, hostPath string, size string) {
 	}
 }
 
+func GetListOfManifests(pathToManifestsDir string) []string {
+	var manifests []string
+	isOpenshift := IsOpenShift()
+	matchFileName := func(pattern, filename string) bool {
+		match, err := filepath.Match(pattern, filename)
+		if err != nil {
+			panic(err)
+		}
+		return match
+	}
+	err := filepath.Walk(pathToManifestsDir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			fmt.Printf("ERROR: Can not access a path %q: %v\n", path, err)
+			return err
+		}
+		if !info.IsDir() {
+			if matchFileName("*-for-ocp.yaml", info.Name()) {
+				if isOpenshift {
+					manifests = append(manifests, path)
+				}
+			} else if matchFileName("*-for-k8s.yaml", info.Name()) {
+				if !isOpenshift {
+					manifests = append(manifests, path)
+				}
+			} else if matchFileName("*.yaml", info.Name()) {
+				manifests = append(manifests, path)
+			}
+		}
+		return nil
+	})
+	if err != nil {
+		fmt.Printf("ERROR: Walking the path %q: %v\n", pathToManifestsDir, err)
+		panic(err)
+	}
+	return manifests
+}
+
+func ReadManifestYamlFile(pathToManifest string) []unstructured.Unstructured {
```

Comment:
Does it work if manifest includes a number of entities?

Reply:
Yes it reads all yaml documents from file.